cmd/program/program.go

```diff
@@ -98,6 +99,9 @@ func (p *Project) CreateMainFile() error {
 		}
 	}
 
+	// Remove trailing whitespaces from ProjectName
```

Comment:
Since the code is self-explanatory right below, this comment is not a value add! We can remove it  :)